src/test/java/core/basesyntax/service/impl/ProductParserImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProductParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProductParserImplTest {
+    private static final String BALANCE_TRANSACTION = "b,banana,20";
+    private static final String PURCHASE_TRANSACTION = "p,banana,13";
+    private static final String RETURN_TRANSACTION = "r,apple,10";
+    private static final String SUPPLY_TRANSACTION = "s,banana,100";
+    private ProductParser productParser;
+
+    @Before
+    public void setUp() {
+        productParser = new ProductParserImpl();
+    }
+
+    @Test
+    public void parseBalanceIs_Ok() {
```

Comment:
<img width="372" alt="Screen Shot 2022-09-14 at 8 20 30 PM" src="https://user-images.githubusercontent.com/90692308/190230240-2ba78280-9393-43fb-a61a-68bebc0b78ad.png">
Use this layout for titles. If you test a class with one method, you don't have to use its name in the beginning. Also, make sure you don't use IS everywhere.